package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "j-arc",
```

Comment:
why this ?

Reply:
Removed this file, will look into how to do this using mvn commands